dev/clint/src/clint/rules/redundant_test_docstring.py

```diff
@@ -0,0 +1,96 @@
+"""Rule to detect redundant docstrings in test functions and classes.
+
+This rule flags single-line docstrings in test functions and classes when:
+1. The docstring is shorter than the function name, AND
+2. More than 50% of the function name's words appear in the docstring
```

Comment:
How effective is the second rule? How many docstrings does it flag?

Reply:
It only flags 24 right now, which is far less than the original single-string rule. I figure it's a good place to start to reduce the chances of flagging docstrings that we don't want to ban quite yet.

Reply:
If it only flags a few, let's remove it.

Reply:
ah I misread your comment. Can we use difflib.SequneceMatcher?